ethereum/vm.py

```diff
@@ -545,10 +591,22 @@ def vm_execute(ext, msg, code):
             if not mem_extend(mem, compustate, op, meminstart, meminsz) or \
                     not mem_extend(mem, compustate, op, memoutstart, memoutsz):
                 return vm_exception('OOG EXTENDING MEMORY')
-            extra_gas = (value > 0) * opcodes.GCALLVALUETRANSFER
+            extra_gas = (value > 0) * opcodes.GCALLVALUETRANSFER + \
```

Comment:
It needs either line continuation or parenthesis:
```python
eval("""1 +
    ...: 2""")
  File "<string>", line 1
    1 +
       ^
SyntaxError: invalid syntax
```
vs
```python
eval("""1 + \
    ...: 2""")
>>> 3
```
There is an argument towards preferring parenthesis over line continuation, but I tend to stick to the established style in this PR.